src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.DataConverter;
+import core.basesyntax.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    private static final String COMA_SEPARATOR = ",";
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputData) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        List<String> dataLines = removeHeaderIfExists(inputData);
+        for (String line : dataLines) {
```

Comment:
General tip - it's better to check if first line is a header, as you did, but then just skip it and continue to iterate through your original `List` of data, because otherwise if file contained hundreds or thousands of lines, you simply allocate a lot of extra memory and copy all it, just to skip first line

Reply:
The changes have been made, but what exactly are you checking: 'jv-fruit-shop-tests' or 'jv-fruit-shop'?